kite-morphlines/kite-morphlines-core/src/test/resources/test-morphlines/hashCommand.conf

```diff
@@ -0,0 +1,73 @@
+# Copyright 2013 Cloudera Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+morphlines : [
+  {
+    id : morphline1
+    importCommands : ["org.kitesdk.**"]
+    
+    commands : [    
+      
+      { logDebug { format : "input record: {}", args : ["@{}"] } }                 
+      
+      { 
+        readBlob {}
+      } 
+      
+      {
+        hashDigest {
```

Comment:
Can we rename hashCommand.conf to hashDigest.conf to be consistent with the previous renames?